### ISSUE TITLE
Using dist/videojs-contrib-hls.js causes a runtime error in the browser.

### ISSUE THREAD

**Issue by iambumblehead** [videojs/videojs-contrib-hls#635]
Using this package with npm/browserify causes a runtime error in the browser. EDIT in fact, copy pasting the content of `dist/videojs-contrib-hls.js` into a browser console causes the runtime error --it is not necessary to use npm or browserify. 

```
Uncaught TypeError: Cannot read property 'EventTarget' of undefined
```

line 10775, seen here, is the source of the error
```
  }]);

  return FlashSourceBuffer;
})(_videoJs2['default'].EventTarget);

exports['default'] = FlashSourceBuffer;
```

**Comment by gkatsev** [comment]
As discussed on IRC, using the `dist` file requires that you add videojs as a global before including contrib-hls.

**Comment by gkatsev** [comment]
Going to close this to continue alternate discussion in #636.
There are two bugs, one is how we export this package (which is fixed by #638) and also the issue in #636 where we always shim videojs.
